optimum/intel/ipex/modeling_base.py

```diff
@@ -325,10 +348,13 @@ def __init__(
     ):
         # Perform the initial warmup at the end of __init__
         super().__init__(model, config, model_save_dir=model_save_dir, warmup=False)
+        GenerationMixin.__init__(self)
 
         self.normalized_config = NormalizedConfigManager.get_normalized_config_class(config.model_type)(config)
         self.model_dtype = kwargs.get("model_dtype", self.dtype)
+        self._dtype = self.model_dtype
         self.use_cache = "past_key_values" in self.input_names
+        self.is_ipex_exported = kwargs.get("is_ipex_exported", None)
```

Comment:
what if we load an already exported model : `export=False` -> `_from_transformers` won't be called, `is_ipex_exported` will always be set to None

Reply:
I think we can check the model's arch to decide if `is_ipex_exported` is True or not, WDYT?

Reply:
Sure, will check if the original_name in the graph model is in our supported list or not.